app/dash/App.js

```diff
@@ -51,7 +51,10 @@ class Dash extends React.Component {
     if (view === 'accounts') return <Accounts data={data} />
     if (view === 'expandedSigner' && data.signer) {
       const signer = this.store('main.signers', data.signer)
-      if (!signer) return null
+      if (!signer) {
+        link.send('tray:action', 'navClearSigner', data.signer)
```

Comment:
shouldn't we do this in `main` when we detect the signer has disconnected?

Reply:
Probably, will update